blockchain/types/transaction_signing_test.go

```diff
@@ -37,6 +37,89 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestPragueSigningWithoutChainID(t *testing.T) {
```

Comment:
How about refactoring the common code in three functions?

Reply:
Thank you for your comment!
It was refoctored by https://github.com/kaiachain/kaia/pull/119/commits/aec484d2a77cc5bc9d28c10f82182cb88b6c84df.